src/beamlime/config/workflow_spec.py

```diff
@@ -179,6 +191,33 @@ class PersistentWorkflowConfigs(BaseModel):
         description="Persistent configurations indexed by workflow ID",
     )
 
+    @field_serializer('configs')
+    def serialize_configs(
+        self, configs: dict[WorkflowId, PersistentWorkflowConfig]
+    ) -> dict[str, Any]:
+        """Serialize configs by converting WorkflowId keys to strings."""
+        return {str(k): v.model_dump() for k, v in configs.items()}
```

Comment:
A while ago I changed `WorkflowId` from a string to a dataclass. I noticed now that this broke the workflow-config-persist mechanism, since Pydantic does not like the new ID type as a dict key in serialization. This fixes this.